.github/workflows/watch-lintdiff.yaml

```
@@ -0,0 +1,21 @@
+# Use ~ to sort the workflow to the bottom of the list
+name: "~Watch - LintDiff"
+
+on:
+  # check_suite is preferred over check_run to avoid triggering on all check
+  # runs. In some cases, check_run must be used in testing environments.
+  check_suite:
+    types: completed
+
+  workflow_run:
+    types: completed
+    workflows: 
+      - "\\[TEST-IGNORE\\] Swagger LintDiff"
+
+jobs:
+  LintDiff:
```

```suggestion
  LintDiff:
  if: ${{ github.event.check_suite.app.name == 'openapi-pipeline-app' }}
```

Add filter on check_suite name, to avoid triggering on other check_suites.  I'm guessing on the syntax, but it should be close to what I have above.